app/client/src/components/Charts/index.tsx

```diff
@@ -19,38 +16,33 @@ const getIconUrl = (chartMetadataName: string, chartGroup: string) => {
 	) {
 		baseURL = `${window.location.protocol}//${window.location.hostname}:8080`;
 	} else baseURL = "/api";
+	if (chartMetadataName === "all-experiments")
```

Comment:
why are we filtering the chart based on name?

Reply:
This is required, when the user goes for all-experiments we do not show an experiment page rather the page which has the link to install all the experiments. To do this we need to check the experiment name.

Reply:
It also has a different design so we have to identify it